cmd/generate/flag.go

```diff
@@ -0,0 +1,41 @@
+package generate
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagApp          = "app"
+	flagDir          = "dir"
+	flagInstallation = "installation"
+	flagVersion      = "version"
+)
+
+type flag struct {
+	App          string
+	Dir          string
+	Installation string
+	Version      string
```

Comment:
I'd name it ConfigVersion and the flag accordingly because it's confusing.

Reply:
Changed.